pepper-apis/src/main/java/org/broadinstitute/ddp/util/ConfigManager.java

```diff
@@ -28,7 +28,16 @@
 
     private static final String TYPESAFE_CONFIG_SYSTEM_VAR = "config.file";
 
-    public static File TYPESAFE_CONFIG_FILE = new File(System.getProperty(TYPESAFE_CONFIG_SYSTEM_VAR));
+    public static final File TYPESAFE_CONFIG_FILE;
+
+    static {
+        // For benefit of GAE. Does not like command line options with "=" characters and env variables with "."
+        String configFileName = System.getenv(TYPESAFE_CONFIG_SYSTEM_VAR.replace('.', '_'));
+        if (configFileName == null) {
+            configFileName = System.getProperty(TYPESAFE_CONFIG_SYSTEM_VAR);
+        }
+        TYPESAFE_CONFIG_FILE = new File(configFileName);
+    }
```

Comment:
This is the other place where we needed to accomodate receiving the parameter from environment rather than from system property. config_file set in GAE *.yaml file